authority/voting/client/client.go

```diff
@@ -246,7 +246,11 @@ func (p *connector) fetchConsensus(auth *config.Authority, ctx context.Context,
 		return nil, err
 	}
 	p.log.Debugf("sending getConsensus to %s", auth.Identifier)
-	cmd := &commands.GetConsensus{Epoch: epoch}
+	cmd := &commands.GetConsensus{
+		Epoch:              epoch,
+		Cmds:               commands.NewPKICommands(p.cfg.PKISignatureScheme),
+		MixnetTransmission: false, // Disable padding for direct dirauth transmission
```

Comment:
Is there a security analysis (read: any at all) that says that we want a special case for padding? Wouldn't it just be safer to chunk it up with padding (if over some limit) or pad it (if under that limit)? Having this be distinguishable seems risky - I think the result is that padding is disabled which implies a distinguisher. Is that correct? Can a third party adversary tell that a client is doing a fetch of a specific thing here?

Reply:
We cannot pad interactions with the PKI. Easier to talk on the phone to explain